package.json

```diff
@@ -3,42 +3,61 @@
   "version": "5.0.5",
   "private": true,
   "dependencies": {
-    "@agrc/dart-board": "^3.2.2",
-    "@agrc/layer-selector": "^4.0.3",
-    "@agrc/sherlock": "^2.2.3",
-    "@arcgis/core": "^4.18.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.32",
-    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@agrc/dart-board": "^3.3.0",
+    "@agrc/layer-selector": "^4.0.4",
+    "@agrc/sherlock": "^2.2.4",
+    "@arcgis/core": "^4.18.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "bootstrap": "^4.5.3",
-    "downshift": "^3.4.8",
+    "bootstrap": "^4.6.0",
+    "downshift": "^6.1.0",
     "lodash.debounce": "^4.0.8",
     "lodash.escaperegexp": "^4.1.2",
     "lodash.sortby": "^4.7.0",
     "lodash.uniqwith": "^4.5.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-error-boundary": "^3.1.0",
-    "react-fast-compare": "^2.0.4",
-    "react-scripts": "^4.0.1",
-    "reactstrap": "^8.8.1",
-    "web-vitals": "^0.2.4"
+    "react-error-boundary": "^3.1.1",
+    "react-fast-compare": "^3.2.0",
+    "react-scripts": "^4.0.3",
+    "reactstrap": "^8.9.0",
+    "web-vitals": "^1.1.1",
+    "workbox-background-sync": "^5.1.4",
+    "workbox-broadcast-update": "^5.1.4",
+    "workbox-cacheable-response": "^5.1.4",
+    "workbox-core": "^5.1.4",
+    "workbox-expiration": "^5.1.4",
+    "workbox-google-analytics": "^5.1.4",
+    "workbox-navigation-preload": "^5.1.4",
+    "workbox-precaching": "^5.1.4",
+    "workbox-range-requests": "^5.1.4",
+    "workbox-routing": "^5.1.4",
+    "workbox-strategies": "^5.1.4",
+    "workbox-streams": "^5.1.4"
```

Comment:
**question**: Do we want all of these in atlas?

Reply:
shoot, that wasn't supposed to end up here. i started this work as a reproduction of the service worker issue.

Reply:
ok rebased that out.